cr.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  Netguru iOS code review task
+//
+
+class paymentViewController: UIViewController {
+    var delegate: PaymentViewControllerDelegate
```

Comment:
`delegate` is not reassigned anywhere in the code, so it should be a constant - let, not var.
It doesn't have a value, so either there must an initialiser that assigns a value to it, or a value should be assigned here.

Reply:
It should also marked private, unless it is accessed by an entity outside the instance.

Reply:
If it can't have a value during the instance init, then it should be marked as var and as an optional:
``` var delegate: PaymentViewControllerDelegate? ```